Simplified/NYPLReadiumViewSyncManager.m

```diff
@@ -137,66 +137,6 @@ - (void)syncReadingPositionForBook:(NSString *)bookID
     NYPLLOG_F(@"serverLocationString %@",serverLocationString);
     NYPLLOG_F(@"currentLocationString %@",currentLocationString);
 
-    NSDictionary *spineItemDetails = weakSelf.bookMapDictionary[responseJSON[@"idref"]];
```

Comment:
this code was cut and pasted down below? no changes at all?

Reply:
No changes, except the `return` to get rid of the `else` bracket